BinaryTree.py

```diff
@@ -0,0 +1,122 @@
+from Lab_10.Resistor import Resistor
```

Comment:
I believe you did all your Python labs in one project (just different folders for lab_9, lab_10 etc.), but you committed the files as-is. If someone wants to clone your repo and run this project, they would get an import error, because Lab_10 package does not exist in your repository.